test/Network/Wai/Handler/HalTest.hs

```diff
@@ -17,5 +19,24 @@ test_ConvertProxyRequest =
     proxyRequest :: ProxyRequest Void <-
       eitherDecodeFileStrict' "test/data/ProxyRequest.json"
         >>= either fail pure
-    waiRequest <- toWaiRequest mempty 443 proxyRequest
-    pure . T.encodeUtf8 $ pShowNoColor waiRequest
+    waiRequest <- toWaiRequest defaultOptions proxyRequest
+    pure . TL.encodeUtf8 $ pShowNoColor waiRequest
+
+test_DefaultBinaryMimeTypes :: TestTree
+test_DefaultBinaryMimeTypes = testCase "default binary MIME types" $ do
```

Comment:
We generally format our code with `ormolu`, which will sadly wreck this nice layout. Can you please reformat? You might like to make a list of tuples if you think that makes a better layout.